EMIP/1-1000/EMIP00164.xml

```diff
@@ -160,6 +221,37 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                        </layout>
                      </layoutDesc>
                   </objectDesc>
+                  <handDesc>
+                    <handNote xml:id="h1" script="Ethiopic">
+                        <seg type="rubrication">
+                        <foreign xml:lang="gez">ማርያም፡</foreign>
```

Comment:
these two foreign elements are obscure to me. what is meant with them?
https://betamasaheft.eu/Guidelines/?q=handnote&id=handDescription

Reply:
This has been the way that we have used to indicate words that are written in red ink, or 'rubricated' - this one is the Ethiopic for 'MAry'